packages/client/components/TeamPromptMeeting.tsx

```diff
@@ -1,27 +1,43 @@
-import styled from '@emotion/styled'
-import React from 'react'
+import React, {Suspense} from 'react'
 import {Link} from 'react-router-dom'
 
-const Root = styled('div')({
-  display: 'flex',
-  alignItems: 'center',
-  justifyContent: 'center',
-  height: '100%'
-})
-
 import logoMarkPurple from '../styles/theme/images/brand/mark-color.svg'
+import MeetingArea from './MeetingArea'
+import MeetingContent from './MeetingContent'
+import MeetingHeaderAndPhase from './MeetingHeaderAndPhase'
+import MeetingStyles from './MeetingStyles'
+import MeetingTopBar from './MeetingTopBar'
+import PhaseHeaderTitle from './PhaseHeaderTitle'
+import PhaseWrapper from './PhaseWrapper'
 
 interface Props {
   meeting: any
 }
 
 const TeamPromptMeeting = (_props: Props) => {
   return (
-    <Root>
-      <Link title='My Dashboard' to='/meetings'>
-        <img alt='Parabol' src={logoMarkPurple} />
-      </Link>
-    </Root>
+    <MeetingStyles>
+      <MeetingArea>
+        <Suspense fallback={''}>
+          <MeetingContent>
+            <MeetingHeaderAndPhase hideBottomBar={true}>
+              <MeetingTopBar
+                avatarGroup={/*:TODO: (jmtaber129): Add avatar group*/<></>}
+                isMeetingSidebarCollapsed={true}
+              >
+                {/* :TODO: (jmtaber129): Add back button */}
+                <PhaseHeaderTitle>Hard-coded standup title</PhaseHeaderTitle>
```

Comment:
I'm not sure if it's the final version of a title, but this component has different styles than what we in the designs for team prompt so I'd just create a new one.

Reply:
Could you point out the style differences for me?  I'm looking through the code + the figma docs, and I'm not seeing any real differences.  Though this specific component's implementation doesn't match the styles as implemented in Figma, it looks like this should still match the styles in the context of other components (e.g. the `MeetingTopBar` has styles that add 16px to the left, etc.)

Reply:
sure, sorry I should have been more specific. 
in figma, on desktop version the title has size of `16px` while `PhaseHeaderTitle` uses `20px` 

```
const PhaseHeaderTitle = styled('h1')({
  fontSize: 16,
  lineHeight: '24px',
  margin: 0,
  padding: 0,
  [meetingTopBarMediaQuery]: {
    fontSize: 20
  }
})
```

not a big deal, but there's a difference 👨‍🎨 😅

Reply:
Ah, it looks like some of the Figma mockups have different font sizes - "Standup MVP, first layout" has font size 20px, while "Initial state, no responses yet" has font size 16px.

I've been under the impression that the intention behind the designs is to make this consistent with existing meeting phase styling, but I don't know for sure.

Lmk what you think, happy to create a new component if that's the way to go 🙂

Reply:
oh, didn't notice designs have different font size, good catch! 👏 
@enriquesanchez @ackernaut what size is correct? 🙏

Reply:
@BartoszJarocki @jmtaber129 Thanks for the ping! It should be 16px 👍